src/axom/slic/core/Logger.hpp

```diff
@@ -257,7 +246,38 @@ class Logger
                   bool filter_duplicates = false);
 
   /*!
-   * \brief Flushes all streams.
+   * \brief Flushes all streams for current rank.
```

Comment:
Here too, it might be good to call out that this call is not collective, and should generally be avoided in favor of  `flushStreams` unless a rank is aborting since it could potentially not play nicely with all `LogStream`s.